README.md

```diff
@@ -1 +1,56 @@
+# Kord
+[![Discord](https://img.shields.io/discord/556525343595298817.svg?color=&label=Kord&logo=discord&style=for-the-badge)](https://discord.gg/6jcx5ev)
+
+This is the GitHub origanization for the Kord project, which aims to provide an idiomatic way to interact with the Discord API with Kotlin
+
+## Projects
+
+Currently we have the following projects
+
+| Name        | Description                                           | Repository                                            |
+| ----------- | ----------------------------------------------------- | ----------------------------------------------------- |
+| kord        | Idiomatic Discord API wrapper                         | [kord](https://github.com/kordlib/kord)               |
+| cache       | Caching library used by Kord                          | [cache](https://github.com/kordlib/cache)             |
+| codegen.kt  | Library we use for generating code                    | [codegen-kt](https://github.com/kordlib/codegen-kt)   |
+| .teamcity   | CI Configuration                                      | [.teamcity](https://github.com/kordlib/.teamcity)     |
+| kordx.emoji | Kotlin emoji library, designed for usage with Discord | [kordx.emoji](https://github.com/kordlib/kordx.emoji) |
+
+## License
+
+Unless stated otherwise in the projects Repository our work is licensed under the [MIT License](https://choosealicense.com/licenses/mit/)
+
+## Code of Conduct
```

Comment:
This should go in a `CODE_OF_CONDUCT.md` file. You can learn more about the files GH supports in [the community health file documentation](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file).

Reply:
This would require us to create this file 5 times and update it 5 times in case we change something, the plan was to link to this file from the separate repos

Reply:
If you think that, then you didn't read that link!